### DISCUSSION TITLE
5.0.3.0

### DISCUSSION THREAD

**Discussion by mgth** [mgth/LittleBigMouse#273] [Announcements]
## What's Changed
* Daemon launch stabilised by @mgth in https://github.com/mgth/LittleBigMouse/pull/271
* setup by @mgth in https://github.com/mgth/LittleBigMouse/pull/272

**Full Changelog**: https://github.com/mgth/LittleBigMouse/compare/v5.0-beta2...v5.0-beta3

<hr /><em>This discussion was created from the release <a href='https://github.com/mgth/LittleBigMouse/releases/tag/v5.0-beta3'>5.0.3.0</a>.</em>

**Comment by agerberoff** [comment]
software doesn't launch
